README.md

```diff
@@ -104,6 +117,11 @@ We realized that, in addition to Spotify, other companies were also using Docker
 
 Hit us up if you're interested in using it or want to collaborate with us. [We're on Discord](https://discord.gg/QmzJApGjyE) or email us at [team@replicate.com](mailto:team@replicate.com).
 
+## Prerequisites
+
+- **macOS or Linux**. Cog works on macOS and Linux, but does not currently support Windows.
+- **Docker**. Cog uses Docker to create a container for your model. You'll need to [install Docker](https://docs.docker.com/get-docker/) before you can run Cog.
+
 ## Install
 
 First, [install Docker if you haven't already](https://docs.docker.com/get-docker/). Then, run this in a terminal:
```

Comment:
This is duplicated now.

Reply:
Yeah that's intentional. I put it in the README and the two getting started guides, so it'll be seen by people regardless of how they end up on the repo. 🤷🏼‍♂️